api/src/main/java/org/eclipse/microprofile/config/inject/ConfigProperty.java

```diff
@@ -139,12 +139,17 @@
     String name() default "";
 
     /**
-     * The default value if the configured property value does not exist. Empty string as the default value will be
-     * ignored, which is same as not setting the default value.
+     * The default value if the configured property property does not exist.
      * <p>
-     * If the target Type is not String a proper {@link org.eclipse.microprofile.config.spi.Converter} will get applied.
-     * That means that any default value string should follow the formatting rules of the registered Converters.
-     *
+     * If the target Type is not String, a proper {@link org.eclipse.microprofile.config.spi.Converter} will get
+     * applied. This value acts as a configure source with the lowest ordinal.
```

Comment:
I feel the sentence _This value acts as a configure source with the lowest ordinal._ got added in a place where it can be misunderstood being linked to the sentence before. Maybe it can be inserted right after the first sentence _The default value if the configured property property does not exist._?

Reply:
+1